Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -72,6 +72,8 @@
     <string name="preference_interoperability_selected_country_codes"><xliff:g id="preference">"preference_interoperability_selected_country_codes"</xliff:g></string>
     <!-- NOTR -->
     <string name="preference_interoperability_all_countries_selected">preference_interoperability_all_countries_selected</string>
+    <!-- NOTR -->
+    <string name="preference_has_risk_status_lowered">preference_has_risk_status_lowered</string>
```

Comment:
Don't use strings.xml for preference keys add a static string variable to `LocalData`

Reply:
Ok, I've implemented a static string in LocalData and removed the string from strings.xml